libs/application-generic/src/usecases/get-subscriber-preference/get-subscriber-preference.usecase.ts

```diff
@@ -33,6 +33,7 @@ export class GetSubscriberPreference {
       {
         organizationId: command.organizationId,
         environmentId: command.environmentId,
+        critical: command.readOnly,
```

Comment:
The `critical` flag is always `false` for code-first Workflows created via the Bridge syncing method, so I believe this implementation will never return code-first defined Workflows when `command.readOnly=true`. Instead, Bridge defined Workflows only use the `PreferenceEntity.preferences.all.readOnly` flag to mark a Workflow as "critical".

Can you please test this and confirm?

If this is the case, we will need to fetch all Workflows (i.e. remove this additional filter on the DB layer) and perform custom filtering using JS to handle both of:
1. V1 Workflows: Workflows who have `critical: true`
2. V2 Workflows: Workflows who have `type: 'BRIDGE', combined with a `PreferenceEntity.preferences.all.readOnly: true`